collect_app/src/main/java/org/odk/collect/android/tasks/InstanceSyncTask.java

```diff
@@ -139,7 +145,13 @@ protected String doInBackground(Void... params) {
                                 values.put(InstanceColumns.DISPLAY_NAME, formName);
                                 values.put(InstanceColumns.JR_FORM_ID, jrFormId);
                                 values.put(InstanceColumns.JR_VERSION, jrVersion);
-                                values.put(InstanceColumns.STATUS, InstanceProviderAPI.STATUS_COMPLETE);
+
```

Comment:
In one or two lines, you can do, with no duplication, what you have in six.
values.put(InstanceColumns.STATUS, instanceSyncFlag ? InstanceProviderAPI.STATUS_COMPLETE : InstanceProviderAPI.STATUS_INCOMPLETE);